app/controllers/dashboard/flickr_controller.rb

```diff
@@ -0,0 +1,14 @@
+require 'flickraw'
+
+class Dashboard::FlickrController < Dashboard::BaseController
+  respond_to :js
+
+  def get_flickr_images
+    FlickRaw.api_key = ENV['FLICKR_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.